peps/pep-0012/pep-NNNN.rst

```diff
@@ -3,7 +3,7 @@ Title: <REQUIRED: pep title>
 Author: <REQUIRED: list of authors' names and optionally, email addrs>
 Sponsor: <name of sponsor>
 PEP-Delegate: <PEP delegate's name>
-Discussions-To: <REQUIRED: URL of current canonical discussion thread>
+Discussions-To: TBA <REQUIRED: URL of current canonical discussion thread>
```

Comment:
I think we almost always want a PEP author to wait until opening the thread. Putting "REQUIRED: URL" will probably still lead to confusion.

Those who know what they're doing can still pre-open a thread, we just don't recommend it.
```suggestion
Discussions-To: TBA
```